core/src/main/java/org/jboss/sbomer/core/features/sbom/utils/UrlUtils.java

```diff
@@ -43,39 +43,26 @@ public static String urldecode(String value) {
     }
 
     /**
-     * Removes from the provided purl the qualifiers that are present (if any) in the allowList. If the purl does not
+     * Removes from the provided purl the qualifiers that are present (if any) in the removeList. If the purl does not
      * contain any qualifier that needs to be removed, the original purl is returned, otherwise a new purl is built and
      * returned.
      *
      * @param purl the purl from which the qualifiers should be removed
-     * @param allowList the list of qualifiers which should be removed from the purl
+     * @param removeList the list of qualifiers which should be removed from the purl
      * @return the purl with the qualifiers removed
      */
-    public static String removeAllowedQualifiersFromPurl(String purl, List<String> allowList) {
+    public static String removeAllowedQualifiersFromPurl(String purl, List<String> removeList) {
```

Comment:
This makes much more sense to me, but some minor nitpicks 

* I don't understand why its `removeAllowedQualifiersFromPurl`  rather than just `removeQualifiersFromPurl` ?

* I think renaming the methods in `UrlUtilsTest.java` for might make tests bit more readable and also helps highlight relevant tests to these changes, they're currently all named "testCaseAllowList" and should probably be "testCaseRemoveList"

Reply:
I agree. I changed the name.